src/adapters/mc/sshpreview/refresh_for_preview.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ -n "$(command -v yum)" ]; then
+        echo "using yum"
+        sudo yum remove openssh-server -y
```

Comment:
i wouldn't put any `sudo` command in the script. Have them invoke with sudo instead. Makes the script much more portable for rootless environments

Reply:
Can you explain why not?

Reply:
As a best practice, scripts should be written to require the least privileges necessary for the task they need to perform. If a script needs to perform actions as root, it’s often better to have the user invoke the script with sudo rather than embedding sudo in the script itself. This makes the need for root privileges explicit and keeps the control in the hands of the user.

Reply:
all of the build containers are rootness, mind you, we use these only for building and not running osconfig. But it is an environment to consider

Reply:
removed sudo from the script